perfkitbenchmarker/linux_benchmarks/ping_benchmark.py

```diff
@@ -103,7 +103,7 @@ def _RunPing(sending_vm, receiving_vm, receiving_ip, ip_type):
   Returns:
     A list of samples, with one sample for each metric.
   """
-  if not sending_vm.IsReachable(receiving_vm):
+  if ip_type == 'internal' and not sending_vm.IsReachable(receiving_vm):
```

Comment:
Why isn't this cased the same as `IpAddressSubset.INTERNAL`

Reply:
It's just a string passed by the Run() function directly above it, but I can change it to INTERNAL/EXTERNAL if that is more consistent

Reply:
Hardcoding the same constant in two different places is not as good as using the same shared constant.  You could define a new one here, but if there's already a enum that does what you want, seems better to use it.

Reply:
Changed to use existing enum